src/script/page/AppMain.tsx

```diff
@@ -125,7 +126,7 @@ const AppContainer: FC<AppContainerProps> = ({root}) => {
           />
 
           {/*The order of these elements matter to show proper modals stack upon each other*/}
-          <div id="user-modal-container"></div>
+          <UserModal actionsViewModel={root.actions} userRepository={repositories.user} />
```

Comment:
I think you can use @przemvs provider to have access to the view models from the component. 
@przemvs can you confirm?

Reply:
https://github.com/wireapp/wire-webapp/pull/13989/commits/d8c9b499f2021a190e2d755df0ab430b2741d258